CadRevealFbxProvider/UserFriendlyLoggerExceptionHandler.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace CadRevealFbxProvider.UserFriendlyLogger;
+
+public class UserFriendlyLoggerExceptionHandler
```

Comment:
This file is a duplicate of UserFriendlyLogger/UserFriendlyLoggerExceptionHandler.cs and should be removed.